src/pages/tasks/common/helpers.ts

```diff
@@ -24,17 +24,101 @@ export function parseTime(timestamp: number) {
     return;
   }
 
-  return dayjs.unix(timestamp).format('hh:mm:ss');
+  const formattedTimestamp = dayjs.unix(timestamp).format('hh:mm:ss A');
+
+  const [timeValues] = formattedTimestamp.split(' ');
+
+  const [hours, minutes, seconds] = timeValues.split(':');
+
+  if (formattedTimestamp.includes('PM')) {
+    const updatedHours =
+      Number(hours) !== 12 ? Number(hours) + 12 : Number(hours);
+
+    return (
+      updatedHours.toString() +
+      ':' +
+      minutes.toString() +
+      ':' +
+      seconds.toString()
+    );
+  }
+
+  const updatedHours = Number(hours) === 12 ? '00' : hours;
+
+  return (
+    updatedHours.toString() +
+    ':' +
+    minutes.toString() +
+    ':' +
+    seconds.toString()
+  );
```

Comment:
Can you elaborate on this? It's a kinda big change so I'd like to know the circumstances which lead to this change.

Reply:
The reason for changing this is actually the time format we need on our time pickers. We need hours in the format 00-23, but we had 00-12. So that's the reason for the change. But, I went to the package documentation and found that for this change we only need to change the string from `hh` to `HH`. So now that's the only change needed.